### ISSUE TITLE
Section component not keyboard accessible

### ISSUE THREAD

**Issue by kburk1997** [YACS-RCOS/yacs#299]
I tried tabbing through the courses page on my local machine, and **could not access any sections using only the keyboard**.

This should be fixed ASAP as this makes YACS unusable for screen reader users and sighted keyboard users.

**Comment by kburk1997** [comment]
This also applies to the schedule view. I suspect it is a bug in the section component, which should be fixed ASAP
